rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,116 @@
+
+#### Data set quality control
+An approach to quality control (QC) is to run a partial or full analysis on a subset of the data before doing a full analysis. This requires defining an analyzable subset of the data to pass on to the processing pipelines. A QC data subset must meet the criteria that allow the QC analysis to run.
+
+## User Stories
+
+As a data contributor, I want to maintain flexibility in the laboratory approaches I can use to perform top-up sequencing, library multiplexing and arrange replicates on flowcells in a way that minimizes overall cost, while being assured that the processing will be correct.
+
+As a data-wrangler, I want to be able to succinctly enter the metadata I receive from the data contributor and have a clear understanding of what I need to do to ensure correct processing.
+
+As a schema developer, I want to ensure we are capturing and requiring, the metadata from a contributor to ensuring the pipelines can run appropriately.
+
+As a member of the pipelines team, I want to ensure that the pipelines process the correct data sets and I can easily access the metadata required to trigger the pipelines.  I do not want to be restricted in designing analysis based on how data was submitted.
+
+As a data consumer, I want to be able to find all data files that need to be processed together so that I can run my data processing pipeline. I do not want to be restricted in designing analysis based on how data was submitted.
+
+As a data consumer, I want to be confident that the HCA DCP has correctly processed all data files that belong together so that I know the matrices I receive have been generated correctly.
+
+As a data consumer, I need to know that a data set is done so that I don't download and use it until it is version-complete.
+
+## Detailed Design
+A new concept of *data group* is added to the DCP data model to address these issues. A data group is defined as a set of specific versions of metadata and data that is complete and consistent by specified criteria. A *data group* is not created until all its contents are complete and submitted to the DSS.  Events are generated when a *data group* is created, updated, or deleted.
+
+A data group has a symbolic scope type that specifies what is represented by the group, as well a the set of criteria by which it is complete.  Most of the above use cases will require a *project submission* scope that indicates submission to a project is complete.
+
+Data groups are implemented as a new bundle type that contains a JSON file listing the FQIDs (UUIDs with versions) of all bundles in the data group. The existing DSS subscription mechanism is used for notifications.  DSS collections were considered as a possible implementation, however they don't support subscriptions and are not indexed, so they are not a sufficient mechanism for implementing *data groups*.
```

Comment:
While I agree with them, IMO the remarks about collections do not aid the clarity of this design.

Reply:
good point, I moved the discussion of why collections were not used to Alternatives.